.github/workflows/workflow.yml

```diff
@@ -0,0 +1,77 @@
+name: Test and Deploy
+on:
+  pull_request:
+    branches: 
+      - master
+  push:
+    branches:
+      - master
+  create:
+    tags:
+      - '*'
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Black
+      run: |
+        pip install black
+        black --check --diff .
+
+  test-extension:
```

Comment:
Can you add a `name:` option and append the python version to the name? Like this: https://github.com/wpilibsuite/allwpilib/pull/2586/files#diff-e9f950f17198d3d5e3122a44230a09b9R23